apps/desktop-wallet/src/components/Buttons/ShortcutButtons.tsx

```diff
@@ -71,6 +72,7 @@ const ShortcutButtons = ({
   const [isAddressOptionsModalOpen, setIsAddressOptionsModalOpen] = useState(false)
 
   const lockWallet = () => {
+    keyring.clearCachedSecrets()
```

Comment:
It deletes all cached secrets, see the implementation later.

Reply:
Good call. I'll look into it.

Reply:
I found a much better solution that combines locking, unlocking, and `isAuthenticated`, all in one hook called `useWalletLock`. Check out 6e4cc0590 in PR #504

Reply:
I like it a lot! 💯